var/spack/repos/builtin/packages/lhapdfsets/package.py

```diff
@@ -50,7 +53,16 @@ def install(self, spec, prefix):
         elif sets == "default":
             default_sets = ["MMHT2014lo68cl", "MMHT2014nlo68cl", "CT14lo", "CT14nlo"]
             sets = default_sets
-        lhapdf("--pdfdir=" + self.prefix.share.lhapdfsets, "install", *sets)
+        with working_dir(self.prefix.share.lhapdfsets):
+            for s in sets:
+                curl(
+                    "-L",
+                    "-o",
+                    "%s.tar.gz" % s,
+                    "http://lhapdfsets.web.cern.ch/lhapdfsets/current/%s.tar.gz" % s,
+                )
+                tar("xfz", s + ".tar.gz")
+                os.remove(s + ".tar.gz")
```

Comment:
Can we gather `s + ".tar.gz"` in a variable, so to avoid mismatches when we delete (which could occur in a refactor)?

Reply:
@alalazo good point, done, thanks!